scripts/kms/js_app_set.sh

```diff
@@ -22,6 +22,10 @@ js-app-set() {
       echo "$line" >> "$temp_file"
     fi
   done < $REPO_ROOT/governance/proposals/set_js_app.json
+
+  # Ensure the target directory exists
+  mkdir -p $WORKSPACE/proposals
```

Comment:
We can revisit this when all other changes are addressed, this shouldn't be necessary and it might hide other issues